src/core/componentSystems/AnimationSystem.js

```diff
@@ -42,8 +42,18 @@ export class AnimationSystem extends MRSystem {
      * @description Called when the entity component is initialized
```

Comment:
can you add `(async)` to the description like you did for the other newly async functions?